app.js

```diff
@@ -1,24 +1,38 @@
-/*jshint es5: true*/
+/**
+ * Main runnable file of Specberus.
+ */
 
-// Pseudo-constants:
-var DEFAULT_PORT = 80;
+// Settings:
+const DEFAULT_PORT = 80;
 
-// The Express and Socket.io server interface
-var express = require("express")
-,   bodyParser = require('body-parser')
+// Native packages:
+const http = require('http');
+
+// External packages:
+const bodyParser = require('body-parser')
 ,   compression = require('compression')
+,   express = require('express')
+,   insafe = require('insafe')
 ,   morgan = require('morgan')
-,   app = express()
-,   server = require("http").createServer(app)
-,   io = require("socket.io").listen(server)
-,   Specberus = new require("./lib/validator").Specberus
-,   l10n = require("./lib/l10n")
-,   util = require("util")
-,   events = require("events")
-,   insafe = require("insafe")
-,   version = require("./package.json").version
+,   socket = require('socket.io')
+;
+
+// Internal packages:
+const package = require('./package.json')
+,   api = require('./lib/api')
+,   l10n = require('./lib/l10n')
+,   sink = require('./lib/sink')
+,   validator = require('./lib/validator')
+;
+
+const app = express()
```

Comment:
Yes, they are. We don't assign a different value to these variables after these declarations.